Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -126,19 +126,9 @@ class DefaultBrowserOnboardingViewController: UIViewController, OnViewDismissabl
         initialViewSetup()
         listenForThemeChange(view)
         applyTheme()
-    }
-
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-        // Portrait orientation: lock enable
-        OrientationLockUtility.lockOrientation(UIInterfaceOrientationMask.portrait,
-                                               andRotateTo: UIInterfaceOrientation.portrait)
-    }
-
-    override func viewWillDisappear(_ animated: Bool) {
-        super.viewWillDisappear(animated)
-        // Portrait orientation: lock disable
-        OrientationLockUtility.lockOrientation(UIInterfaceOrientationMask.all)
+        if #available(iOS 16, *) {
```

Comment:
same here  you can use a conditional statement in a single line using the nil coalescing operator
single-line expression are more concise and may be preferred for simple availability checks where you want to keep the code short
#available(iOS 16, *) ? setNeedsUpdateOfSupportedInterfaceOrientations() : ()